static/search.less

```diff
@@ -78,7 +78,7 @@
 		max-width: 350px;
 	}
 	max-height: calc(~"100vh - 73px");// 53px header + 20px drop shadow
-	z-index: 999998;
+	z-index: @z-index-top;
```

Comment:
This one doesn’t seem necessary either because the header already has a z-index.